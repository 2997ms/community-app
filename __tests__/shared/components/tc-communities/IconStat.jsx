import React from 'react';
import Rnd from 'react-test-renderer/shallow';
import IconStat from 'components/tc-communities/IconStat';

const rnd = new Rnd();

function Icon() {
  return <div />;
}

test('Snapshot match', () => {
  rnd.render((
    <IconStat icon={Icon} number="5" label="Projects" />
  ));
  expect(rnd.getRenderOutput()).toMatchSnapshot();

  rnd.render((
    <IconStat
      icon={Icon}
      number="5"
      label="Projects"
      theme={{
        container: 'container',
        icon: 'icon',
        number: 'number',
        label: 'label',
      }}
    />
  ));
  expect(rnd.getRenderOutput()).toMatchSnapshot();
});
