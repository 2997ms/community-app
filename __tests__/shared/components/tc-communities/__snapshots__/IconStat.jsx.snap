// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot match 1`] = `
<div
  className="container src-shared-components-tc-communities-IconStat-___style__container___145TJ"
>
  <div />
  <div
    className={undefined}
  >
    <p
      className="number src-shared-components-tc-communities-IconStat-___style__number___3GKcu"
    >
      5
    </p>
    <p
      className="label src-shared-components-tc-communities-IconStat-___style__label___3v7Pl"
    >
      Projects
    </p>
  </div>
</div>
`;
